### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./.git,./docs/old,./old_nodes -L bu,indicies,nd,nin,pullrequest,ro,splitted,vertexes`

### CODE REVIEW

File: docs/nodes/list_main/match.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Parameters
 * **Short**: cut all to minimal length of lists
 * **Cycle**: extend all to maximal length of lists by cyclingly repeating
 * **Repeat**: extend all to maximal length of lists by last item repeating
-* **X-Ref**: extend all to maximal length of lists by multiply lengthes,
+* **X-Ref**: extend all to maximal length of lists by multiply lengths,
```
Comment (luzpaz) [inline docs/nodes/list_main/match.rst:37]: Perhaps "...by muliply**ing** lengths" was meant here?

File: nodes/pulga_physics/pulga_boundaries_force.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 class SvPulgaBoundingBoxForceNode(SverchCustomTreeNode, bpy.types.Node):
     """
-    Triggers: Spacial Ambit
+    Triggers: Spatial Ambit
```
Comment (luzpaz) [inline nodes/pulga_physics/pulga_boundaries_force.py:35]: Please confirm

File: docs/nodes/list_main/match.rst (hunk 1 above)
Comment (Durman) [reply docs/nodes/list_main/match.rst:37]: It seems so

File: nodes/pulga_physics/pulga_boundaries_force.py (hunk 2 above)
Comment (Durman) [reply nodes/pulga_physics/pulga_boundaries_force.py:35]: @vicdoval

File: docs/nodes/list_main/match.rst (hunk 1 above)
Comment (luzpaz) [reply docs/nodes/list_main/match.rst]: Fixed in [2c0c522](https://github.com/nortikin/sverchok/pull/4900/commits/2c0c5224fa151524a4007ce2596a0c3afafc9001)

File: nodes/pulga_physics/pulga_boundaries_force.py (hunk 2 above)
Comment (luzpaz) [reply nodes/pulga_physics/pulga_boundaries_force.py:35]: bump

**Pull request merged**